docs/root/intro/version_history.rst

```diff
@@ -127,6 +127,8 @@ Version history
 * stats: added support for histograms.
 * stats: added :ref:`option to configure the statsd prefix<envoy_api_field_config.metrics.v2.StatsdSink.prefix>`
 * stats: updated stats sink interface to flush through a single call.
+* tls: added support for allowing expired certificates with
```

Comment:
Nit: could you move it to after `verify_certificate_*` entries but before `allow_renegotiation`?

Reply:
Sure